src/outputs/map_output_cols.py

```diff
@@ -64,6 +64,11 @@ def join_fgn_ownership(
         )
         combined_df.drop(columns=["ruref"], inplace=True)
 
+        # If foreign ownership is empty, we fill it with "GB" for long, short 
+        # and NI
+        combined_df["ultfoc"] = combined_df["ultfoc"].fillna("GB")
+
+
```

Comment:
If NI needs to be filled GB for NI as well, does this need to go after the concat in line 72?

Reply:
Or can the split of 0003 from 0001/0006 be taken out entirely?